python/suricata/ctl/filestore.py

```diff
@@ -82,22 +82,32 @@ def remove_file(path, dry_run):
     return size
 
 
-def prune(args):
+def set_logger_level(args):
     if args.verbose:
         logger.setLevel(logging.DEBUG)
     if args.quiet:
         logger.setLevel(logging.WARNING)
 
-    if not args.directory:
-        print(
-            "error: the filestore directory must be provided with --directory",
-            file=sys.stderr)
-        return 1
 
-    if not args.age:
-        print("error: no age provided, nothing to do", file=sys.stderr)
-        return 1
+def perform_sanity_checks(args):
+    set_logger_level(args)
```

Comment:
This should probably be in the prune() method, but we can fix that up another time.